tests/nox/conftest.py

```diff
@@ -0,0 +1,68 @@
+"""This file is only for the database fixture. For all other fixtures add them to the
+tests/conftest.py file.
+"""
+
+import pytest
```

Comment:
I think we can trash this file and have a pytest command look something like `pytest tests/nox/ --noconftest`